src/main/resources/META-INF/resources/js/filemanager.js

```diff
@@ -2530,60 +2530,66 @@ function fileManager(
     var $thisLink = $(this)
 
     if ($thisLink.attr('class') === 'download-zip-file') {
-      var postData = {}
-      postData.responseformat = 'application/zip'
-      var targetList = new Array();
 
-      $.each(
-        $dt
-          .rows({
-            selected: true
-          })
-          .data(),
-        function (key, itemData) {
-          targetList.push(itemData[10])
-        }
-      )
+      // CADC-10355: zip download only available for vault.
+      // if the element is disabled, do not attempt to post request.
+      if ($thisLink.attr('disabled') !== 'disabled') {
```

Comment:
This is a little fragile.  Does `if ($thisLink.prop('disabled'))` work?  It's valid to set `disabled='true'`.